### ISSUE TITLE
several  questions

### ISSUE THREAD

**Issue by zhenmafan7** [jfzhang95/DeepGrabCut-PyTorch#3]
Hi,thanks for you provide this project,and several questions make me confused:
1.while training,what is the GT images?The bounding-box annotation images?Or the mask annotation images?Or no GT images?
2.The input images is the bounding-box annotation images,right?where can I get the dataset?
3.In the paper,it said that this method can accomplish semantic/instance segmentation,but this project only can realized object foreground segmentation.Is there any idea you could provide me to realize semantic segmentation?
 
Sincerely look forward to your reply.

**Comment by ty199931** [comment]
> Hi,thanks for you provide this project,and several questions make me confused: 1.while training,what is the GT images?The bounding-box annotation images?Or the mask annotation images?Or no GT images? 2.The input images is the bounding-box annotation images,right?where can I get the dataset? 3.In the paper,it said that this method can accomplish semantic/instance segmentation,but this project only can realized object foreground segmentation.Is there any idea you could provide me to realize semantic segmentation?
> 
> Sincerely look forward to your reply.

你好，我也想问一下，这个的数据集是怎么得到的？请问您弄清楚了吗？可以给我讲讲吗？
